### DISCUSSION TITLE
[Esame 20 Luglio 2021] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#261] [Esami - Bertini @ Matematica]
![Screenshot 2023-11-14 113101](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/1b3aa977-6ae6-42bf-b125-01214a2198bc)

**Comment by CasuFrost** [comment]
#### punto $a$ )
Se con "legge" si intende, la funzione di distribuzione, la risposta è no. La legge che descrive $T_i$ è 
$$1-(1+\lambda t)e^{-\lambda t}$$
#### punto $b$ )
La funzione densità è una primitiva della funzione di distribuzione : 
$$\dfrac{d}{dt}[1-[(1+\lambda t)e^{-\lambda t}]=-\lambda e^{-\lambda t}+\lambda (1+\lambda t)e^{-\lambda t}$$
#### punto $c$ )
Il valore atteso equivale a : 
$$\mathbb{E}(T_i)=\displaystyle\int_{-\infty}^{+\infty}t\cdot[-\lambda e^{-\lambda t}+\lambda (1+\lambda t)e^{-\lambda t}]dt$$
Si svolge per parti : 
$$\mathbb{E}(T_i) = [1-(t+\lambda t)e^{-\lambda t}]\cdot t - \displaystyle\int_{-\infty}^{+\infty}[-\lambda e^{-\lambda t}+\lambda (1+\lambda t)e^{-\lambda t}]dt =  [1-(t+\lambda t)e^{-\lambda t}]\cdot t -  [1-(1+\lambda t)e^{-\lambda t}]$$
$$\implies \mathbb{E}(T_i)=(t-1)[1-(1+\lambda t)e^{-\lambda t}]$$
La varianza è $\mathbb{V}(T_i)=\mathbb{E}(T_i^2)-\mathbb{E}(T_i)^2$, esplicito : 
$$\mathbb{V}(T_i)=\displaystyle\int_{-\infty}^{+\infty}t^2\cdot[-\lambda e^{-\lambda t}+\lambda (1+\lambda t)e^{-\lambda t}]dt-[(t-1)(1-(1+\lambda t)e^{-\lambda t})]^2$$
Si risolve per parti : 
$$(1-(1+\lambda t)e^{-\lambda t})t^2-2\displaystyle\int_{-\infty}^{+\infty}t\cdot[-\lambda e^{-\lambda t}+\lambda (1+\lambda t)e^{-\lambda t}]dt-[(t-1)(1-(1+\lambda t)e^{-\lambda t})]^2$$
$$=(1-(1+\lambda t)e^{-\lambda t})t^2-2\mathbb{E}(T_i)-[(t-1)(1-(1+\lambda t)e^{-\lambda t})]^2$$
Skippo alcuni passaggi algebrici basilari ed ottengo : 
$$\mathbb{V}(T_i)=(1-(1+\lambda t)e^{-\lambda t})(t^2-2t+2)-[(t-1)(1-(1+\lambda t)e^{-\lambda t})]^2$$
#### punto $d$ ) 
Non mi va
